index.d.ts

```diff
@@ -5,6 +5,7 @@
 type rules = object;
 
 type result = {
+  timeout: boolean;
   perfData?: string;
   perfTotalRuntime?: number;
   data?: string;
```

Comment:
`action` has been removed no ?

Reply:
no, it is still populated based on the return code :)